dojson/contrib/marc21/fields/ad01x09x.py

```diff
@@ -386,6 +400,11 @@ def library_of_congress_call_number(self, key, value):
 @utils.filter_values
 def geographic_classification(self, key, value):
     """Geographic Classification."""
+    indicator_map1 = {
+        "_": "Library of Congress Classification",
+        "1": "U.S. Dept. of Defense Classification",
+        "7": "Source specified in subfield $2",
```

Comment:
please see an example how to handle indicator `7` in other fields - e.g. `dojson/contrib/marc21/fields/bd84188x.py` - `location/852`.